File: bubblesort/bubblesort.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){
    int lista[1000];
    int n;
    cin >> n;
    for(int i=0;i<n;i++){
        cin >> lista[i];
    }
    for(int i=0;i<n-1;i++){
        for(int i=0;i<n-1;i++){
            if(lista[i]>lista[i+1]){
                int temp = lista[i];
                lista[i]=lista[i+1];
                lista[i+1]=temp;
            }
        }
    }
    for(int i=0;i<n;i++){
        cout<<lista[i]<<endl;
    }
}

File: bubblesort/bubblesort.py
lista = []
i = int(input())
for i in range(0,i):
    p = int(input(f"{i}"))
    lista.append(p)
print(lista)
j = 0;
k = 0;
for j in range(0, i):
    for k in range(0,i-1):
        if(lista[k]>lista[k+1]):
            temp = lista[k]
            lista[k] = lista[k+1]
            lista[k+1] = temp
            
print(lista)